test/cookbooks/test/templates/package.json

```diff
@@ -10,9 +10,13 @@
     "node": ">=0.10.0"
   },
   "dependencies": {
+    "config": "^1.26.2",
     "koa": "^1.1.0"
   },
   "devDependencies": {
     "gulp": "^3.9.0"
+  },
+  "scripts": {
+    "postinstall": "touch $NODE_ENV"
```

Comment:
@MarkGibbons so this ½ of the test I wrote to ensure this works. As part of the `postinstall`, it attempts to a file named the same as `$NODE_ENV`.